pkg/fakerp/customer_handlers.go

```diff
@@ -109,83 +110,110 @@ func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
 }
 
 func (s *Server) handleGet(w http.ResponseWriter, req *http.Request) {
-	s.reply(w, req)
+	s.customerReply(w, req)
 }
 
-func (s *Server) handlePut(w http.ResponseWriter, req *http.Request) {
+func (s *Server) handleAdminPut(w http.ResponseWriter, req *http.Request) {
```

Comment:
I think I'm -1 on splitting handlePut -> handle{Admin,Customer}Put.  I think refactors can be done to neaten handlePut instead.  Is there a good reason for splitting that I'm missing?